test/object.js

```diff
@@ -1,4 +1,5 @@
 "use strict";
+/* global BigInt */
```

Comment:
IMO, the `ecmaVersion` should rather be updated in the `.eslintrc.json` rather than adding inlining a global.

Reply:
`ecmaVersion`&nbsp;controls which&nbsp;syntax **ESLint** will&nbsp;consider&nbsp;valid, not&nbsp;which&nbsp;globals are&nbsp;defined.

Reply:
I’ve&nbsp;added&nbsp;`BigInt` to&nbsp;`overrides` for&nbsp;tests in&nbsp;`.eslintrc.json`: https://github.com/jsdom/webidl-conversions/blob/9255ff9c7cfbdd566c01bab457df440168faf125/.eslintrc.json#L259-L268